parkhangs-webapp/src/components/EventForm.js

```diff
@@ -84,15 +85,25 @@ class EventForm extends React.Component {
         const eventTimestamp = moment(`${this.parsedEventTime()} ${this.parsedEventDate()}`, 'hh:mm D MM YY').unix()
 
         const detail =  this.state.eventDetail || this.eventDetail();
-        const eventDateTime =  eventTimestamp;
+        const eventDateTime = eventTimestamp
 
-        // ADD/UPDATE EVENT DISPATCH GOES HERE!
         if (!this.props.eventId) {
-           this.props.addOneEvent(this.props.parkId, detail, eventDateTime)
+            const newEvent = {
+                parkId: this.props.parkId,
+                details: detail,
+                eventDateTime,
+            }
+            this.props.addOneEvent(newEvent)
         }
 
         else {
-            this.props.updateEvent(this.props.eventId, this.props.parkId, detail, eventTimestamp)
+            const updatedEvent = {
+                eventId: this.props.eventId,
+                parkId: this.props.parkId,
+                details: detail,
+                eventDateTime,
+            }
+            this.props.updateEvent(updatedEvent)
             this.props.eventChanged(this.props.eventId, detail, eventTimestamp)
```

Comment:
Are we refactoring this out? Because we don't want to pass the event object back to the parent component.

Reply:
I'm going to delete this entirely when I refactor this modal to get the events from the store rather than having it passed from the parent component to the modal